src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -141,8 +142,18 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
     if (detectedFieldsFrame) {
       this.updateOptions(detectedFieldsFrame);
     }
+  }
+
+  private getPrimaryLabel(): RouteProps {
+    let { breakdownLabel, labelName, labelValue } = getRouteParams(this);
```

Comment:
We could probably check for a label value here. Probably if labelValue is missing or empty it would generate an error state.

Reply:
Good catch: although we wouldn't be be on this route if we didn't have a full label/value pair, but we could have a mismatch if someone copied partial url params or something weird like that? Fixed in https://github.com/grafana/logs-drilldown/pull/1294/commits/ea826f73b8d13eb07e1edfcb4b52a2cd2a841117